src/Livewire/Settings/MailAccounts.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace FluxErp\Livewire\Settings;
+
+use FluxErp\Actions\MailAccount\CreateMailAccount;
+use FluxErp\Actions\MailAccount\DeleteMailAccount;
+use FluxErp\Actions\MailAccount\UpdateMailAccount;
+use FluxErp\Livewire\DataTables\MailAccountList;
+use FluxErp\Livewire\Forms\MailAccount as MailAccountForm;
+use FluxErp\Models\MailAccount;
+use Illuminate\Validation\ValidationException;
+use Livewire\Attributes\Renderless;
+use Spatie\Permission\Exceptions\UnauthorizedException;
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+use TeamNiftyGmbH\DataTable\Htmlables\DataTableButton;
+use Webklex\PHPIMAP\Exceptions\AuthFailedException;
+use Webklex\PHPIMAP\Exceptions\ConnectionFailedException;
+use Webklex\PHPIMAP\Exceptions\ImapBadRequestException;
+use Webklex\PHPIMAP\Exceptions\ImapServerErrorException;
+use Webklex\PHPIMAP\Exceptions\ResponseException;
+use Webklex\PHPIMAP\Exceptions\RuntimeException;
+
+#[Renderless]
+class MailAccounts extends MailAccountList
+{
+    protected string $view = 'flux::livewire.settings.mail-accounts';
+
+    public MailAccountForm $mailAccount;
+
+    public function getTableActions(): array
+    {
+        return [
+            DataTableButton::make()
+                ->label(__('Create'))
+                ->color('primary')
+                ->icon('plus')
+                ->attributes([
+                    'x-on:click' => '$wire.edit()',
```

Comment:
does this work? edit method required 1 parameter

Reply:
dependency injection adds an empty model if no param is set